components/user/UserTestimonials.js

```diff
@@ -12,14 +12,9 @@ export default function UserTestimonials({ users, data, BASE_URL }) {
           url: `${BASE_URL}/${testimonial.username}`,
         };
       }
-
-      return {
-        ...testimonial,
-        url: `https://github.com/${testimonial.username}`,
```

Comment:
Not sure why this was removed?

Reply:
It was no longer needed. As, the url property of the testimonial object is being set to `${BASE_URL}/${testimonial.username}`, which appears to be the desired value for the url property. And, the `return testimonial;` statement is being used as the default return value.

Reply:
It will use their LinkFree profile but if it doesn't exist it will default to their GitHub profile, so this is still needed

Reply:
ok then!

Reply:
Great 👍 when this is put back in, I think we can merge this

Reply:
I have updated and checked the code, it is ready for review and merge. Please let me know if you have any questions or concerns about the changes.